Csv/CsvReader.cs

```diff
@@ -135,7 +135,20 @@ private static IEnumerable<ICsvLine> ReadImpl(TextReader reader, CsvOptions opti
                         continue;
                 }
 
-                // TODO: #11 might need to read another line if this one isn't ended yet (i.e. open quoted value)
+                if (options.AllowNewLineInEnclosedFieldValues)
+                {
+                    var separator = Regex.Escape(options.Separator.ToString());
+                    // if there are an odd number of quotes, the value continues on the next line
+                    while (Regex.Matches(line, $@"^""|{separator}""|""{separator}|""$").Count % 2 == 1)
```

Comment:
Actually, we only care about quotes at the beginning or end of field values...  This should not break the non-standard `'escaped"value'` case.

Reply:
but it will break `"escaped""\r\nvalue"`... :(
I just pushed some changes that should make all of that work even better.